cdk-testsuite/libraries/openshift/openshift.py

```diff
@@ -0,0 +1,251 @@
+'''
+Created on Jun 29, 2016
+
+@author: amit
+'''
+from avocado.utils import process
+import imp
+import logging
+import re
+import time
+
+log = logging.getLogger("Openshift.Debug")
+
+
+def openshiftLibInfo(self):
+    '''
+    Get openshift user defined library version
+    Args:
+        self (object): Object of the current method
+    '''
+    openshiftUtils = imp.load_source('openshiftUtils', self.params.get('openshift_util_MODULE'))
+    self.log.info("Openshift library version : " +openshiftUtils.get_version())
+    
+def oc_usr_login(self, ip_port, uname, password):
+    '''
+    Login to openshift server and returns output of the oc command executed
+    Args:
+        self (object): Object of the current method
+        ip_port (str): ip and port of openshift to be used
+        uname (str): username of openshift web console to be used
+        password (str): password of openshift web console to be used
+    '''
+    strcmd = "vagrant ssh -c 'oc login " +ip_port +" --username=" +uname +" --password=" +password +" --insecure-skip-tls-verify" +"'"
+    self.log.info ("Executing : " +strcmd)
+    output = process.system_output(strcmd)
+    return output
+
+def add_new_project(self, project_name):
+    '''
+    Adding new project to openshift server and returns output of the oc command executed
+    Args:
+        self (object): Object of the current method
+        project_name (str): name of the project to be added to the openshift server
+    '''
+    strcmd = "vagrant ssh -c 'oc new-project " +project_name +"'"
+    self.log.info ("Executing : " +strcmd)
+    time.sleep(2)
+    output = process.system_output(strcmd)
+    return output

```

Comment:
I think it would be better to return just true / false and handle error logging in library